backend/src/etc/clean-words.py

```diff
@@ -0,0 +1,25 @@
+# Adam Fernandes
+# ShellHacks 2020
+# Quick and dirty script for cleaning previous data
+
+# Read file
+fileObject = open("keywords.txt", "r")
```

Comment:
So I actually just merged my PR which shows how you want to open files in Python using "with" + try/except to catch any errors

Here is an example: https://github.com/Hevia/JusticeBytes/blob/master/backend/src/etc/FileHelpers.py#L11